src-backend/api/serializer.py

```diff
@@ -44,11 +44,30 @@ def to_representation(self, instance):
         return ret
 
 
+class PageListSerializer(serializers.ListSerializer):
+    class Meta(object):
+        model = models.Page
+
+    def update(self, instance, validated_data):
+        page_mapping = {page.id: page for page in models.Page.objects.all()}
+        data_mapping = {item['id']: item for item in validated_data}
+
+        res = []
+        for page_id, data in data_mapping.items():
+            page = page_mapping.get(page_id, None)
+            if page is not None:
+                res.append(self.child.update(page, data))
+
+        return res
+
+
 class PageSerializer(serializers.ModelSerializer):
     elements = ElementSerializer(many=True, read_only=True)
+    id = serializers.IntegerField(read_only=False)
```

Comment:
Can we make sure that this isn't being changed somehow by the user or at least open a ticket for it?